atd-vze/src/views/Crashes/Crash.js

```diff
@@ -137,6 +134,9 @@ function Crash(props) {
     address_primary: primaryAddress,
     address_secondary: secondaryAddress,
     crash_injury_metrics_view: { years_of_life_lost: yearsOfLifeLost },
+    investigator_narrative: investigatorNarrative,
+    cr3_stored_flag: cr3StoredFlag,
+    cr3_file_metadata: cr3FileMetadata,
   } = crashData?.crashes_by_pk ? crashData?.crashes_by_pk : {};
```

Comment:
this is what i was mentioning in dev sync, that the nested entries will error out when its the fallback `{}` .

Reply:
thanks for flagging this Chia! i have a fix staged in my ID column branch.